2. Web API now calls Microsoft Graph/TodoListService/Startup.cs

```diff
@@ -45,7 +45,7 @@ public Startup(IConfiguration configuration)
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddProtectWebApiWithMicrosoftIdentityPlatformV2(Configuration)
-                    .AddProtectedApiCallsWebApis(Configuration, new string[] { "user.read" })
+                    .AddProtectedApiCallsWebApis(Configuration, null /*new string[] { "user.read" }*/)
```

Comment:
If you don't specify scopes, then the OBO will be lazy

Reply:
like = scopes = null  in the definition, @bgavrilMS  ?

Reply:
I don't understand the question @jmprieur - I was referring to having 2 methods

.AddProtectApiCallsWebApisOnDemand
.AddProtectApiCallsWebApis

Reply:
@bgavrilMS : I rather used a default parameter (scopes = null) => if the scopes are not specified, the obo is delayed until the Web API calls a downstream API (for the user)